ctapipe/image/extractor.py

```diff
@@ -1025,38 +1025,43 @@ def _apply_second_pass(
             return charge_1stpass, pulse_time_1stpass
 
         # otherwise we proceed by parametrizing the image
-        hillas = hillas_parameters(camera_geometry, image_2)
+        hillas = hillas_parameters(camera_geometry[mask_1], charge_1stpass[mask_1])
```

Comment:
Also here mask_biggest is used

Reply:
Ah, in that case we can compute the masked version only once - I'll change it (right now it's sliced two time)